mkdocs/utils.py

```diff
@@ -299,10 +300,23 @@ def convert_markdown(markdown_source, extensions=None, extension_configs=None):
     return (html_content, table_of_contents, meta)
 
 
+def get_themes():
+    """Return a dict of theme names and their locations"""
+
+    themes = {}
+
+    for theme in iter_entry_points(group='mkdocs.themes', name=None):
+        theme = theme.load()
+        location = os.path.dirname(theme.__file__)
+        themes[theme.name] = location
```

Comment:
I expect this is failing because you are replacing the entry_point (var named `theme`) with the loaded object. Before `theme = theme.load()` do `name = theme.name` or don't overwrite the `theme` var (load to a different var).